azure-pipelines.yml

```diff
@@ -65,8 +65,121 @@ stages:
+            Python38:
+              python.version: '3.8'
+        steps:
+          - checkout: self
+            submodules: true
+          - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+            displayName: Add conda to PATH
+          - bash: |
+              set -x
+              set -e
+              conda create --yes --quiet --name qiskit-aer python=$(python.version)
+              source activate qiskit-aer
+              conda update --yes -n base conda
+              conda config --add channels conda-forge
+              conda install --yes --quiet --name qiskit-aer python=$(python.version) numpy cmake openblas virtualenv pip setuptools pybind11 cython
+            displayName: Create Anaconda environments
+          - bash: |
+              set -x
+              set -e
+              source activate qiskit-aer
+              python setup.py bdist_wheel -- -G "Visual Studio 15 2017 Win64"
+            displayName: Build wheels
+          - bash: |
+              set -x
+              set -e
+              source activate qiskit-aer
+              virtualenv test-wheel
+              test-wheel/Scripts/pip install dist/*whl
+              test-wheel/Scripts/pip install git+https://github.com/Qiskit/qiskit-terra
+              test-wheel/Scripts/python tools/verify_wheels.py
+            displayName: Verify wheels
+      - job: 'Windows_sdist_Builds'
+        pool: {vmImage: 'vs2017-win2016'}
+        condition: not(startsWith(variables['Build.SourceBranch'], 'refs/tags'))
+        strategy:
+          maxParallel: 1
+          matrix:
+            Python37:
+              python.version: '3.7'
+            Python38:
+              python.version: '3.8'
+        steps:
+          - checkout: self
+            submodules: true
+          - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+            displayName: Add conda to PATH
+          - bash: |
+              set -x
+              set -e
+              conda create --yes --quiet --name qiskit-aer python=$(python.version)
+              source activate qiskit-aer
+              conda update --yes -n base conda
+              conda config --add channels conda-forge
+              conda install --yes --quiet --name qiskit-aer python=$(python.version) numpy cmake openblas virtualenv pip setuptools pybind11 cython
+            displayName: Create Anaconda environments
+          - bash: |
+              set -x
+              set -e
+              source activate qiskit-aer
+              python setup.py sdist -- -G "Visual Studio 15 2017 Win64"
+            displayName: Build sdist
+          - bash: |
+              set -x
+              set -e
+              source activate qiskit-aer
+              pip install dist/*tar.gz
+              pip install git+https://github.com/Qiskit/qiskit-terra
+              python tools/verify_wheels.py
+            env:
+              CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
```

Comment:
Is this needed to install from sdist? If so we need to document this for installing from sdist via pip somewhere.

Reply:
This is more like a corner-case. It's one of the options for someone on Windows with several toolchains installed on it, and not familiar with MSVC workflow (it usually requires running a batch file that setups some environment variables beforehand, so it becomes the default one).
We don't support other compilers on Windows, but we could certainly add something in the doc.